databricks/koalas/spark/accessors.py

```diff
@@ -851,6 +903,58 @@ def apply(self, func, index_col: Optional[Union[str, List[str]]] = None) -> "ks.
             )
         return output.to_koalas(index_col)
 
+    @property
+    def analyzed(self) -> "ks.DataFrame":
```

Comment:
@ueshin, should we maybe just expose Spark's `checkpoint` or `localCheckpoint` instead?

Reply:
As it will invoke another Spark job, I think we should have them separately.